src/components/WwwFrame/TheFooter.vue

```diff
@@ -36,6 +36,18 @@
 					Terms of use
 				</router-link>
 			</div>
+			<ul class="hide-for-print download-app-icons">
+				<li class="app-store-icon">
+					<a :href="appStoreUrl" target="_blank">
+						<img src="@/assets/images/app-store.svg">
```

Comment:
It would be good to add alt text to these images since there's no text inside the link, maybe `alt="Download Kiva's App on the App Store"`

Also, since these are on the footer we can add [`loading="lazy"` ](loading="lazy" )attribute to prevent them from being downloaded on Chrome unless they're in view.

Reply:
Thanks for the headsup @ryan-ludwig